36-axolotl/technical_specification.md

```diff
@@ -0,0 +1,81 @@
+# Biomedical Metadata Mocking (Axolotl)
+**Epic Type:** Implementation Epic
+
+## Scope
+### Outline:
+This epic implements mocks for all Metadata Repository Service and Metadata Search Service API calls consumed by the Data Portal UI.
+
+### Included/Required:
+The epic includes and requires:
+- the addition of data objects to be mocked based on the current metadata model and currently used metadata objects in the deployed version
+- that there is at least one data object for each edge case (e.g. at least one object with/without an EGA Accession ID (vs. a GHGA Accession ID), a DAC form, a linked study)
+
+### Not included:
+This epic does not include:
+- the addition of 'realistic' data; the data will be used to test what is necessary to test the display of all features available on the data portal, thus additional fields that are not used will not be included.
+
+## User Journeys (optional)
+
+This epic covers the following user journeys:
+
+Browsing data:
+- The user views the summary of the entire dataset of metadata objects
+- The user views the summary of a single metadata object
+- The user views the details of a single metadata object, including list of files, experiments, and samples
+- The user requests access to the data of a specific metadata object through the "Request" access button in either the browse page or the single dataset view page
+- The user filters datasets by keyword
+- The user filters datasets by filter facets
```

Comment:
```suggestion
- The user filters datasets by keyword
- The user filters datasets by filter facets

Please note, filtering and specifying keywords will not change the displayed items since the response from the metadata repository service is mocked and static. This should also be documented in the readme.
```